src/components/SharedComponents/InlineUser.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { Image, Text, View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import IconMaterial from "react-native-vector-icons/MaterialIcons";
+import User from "realmModels/User";
+import colors from "styles/tailwindColors";
+
+type Props = {
+  user: any,
+};
+
+const InlineUser = ( { user }: Props ): Node => {
+  const userImgUri = User.uri( user );
+  const username = `@${user.login}`;
+  return (
+    <View className="flex flex-row items-center">
+      {userImgUri ? (
+        <Image className="w-6 h-6 rounded-full mr-1.5" source={userImgUri} />
```

Comment:
`w-6` and `h-6` both correlate to `width: 1.5rem; /* 24px */` on the tailwinds docs. Closest to the 22px spec.